retrieval-market.md

```diff
@@ -63,14 +77,16 @@ type CidPrefix struct {
 }
 ```
 
+
+
 `Retrieval miners` should also support a query service that allows clients to request pricing information from a miner.
 
 The query should include the CID of the piece that the client is interested in retrieving. The response contains whether or not the miner will serve that data, the price they will accept for it.
 
 ```sh
 type RetQuery struct {
     ## TODO: what exactly does this link to?
-	piece Link
+	piece Cid
```

Comment:
This should also allow specifying `RetParams`. (In case of unixfs we should also define that if `size==0`, then the clients wants the entire thing)